es-app/src/VolumeControl.cpp

```diff
@@ -260,6 +295,17 @@ int VolumeControl::getVolume() const
 #elif defined(__linux__)
 	if (mixerElem != nullptr)
 	{
+		if (mixerHandle != nullptr)
+			snd_mixer_handle_events(mixerHandle);
+		/*
+		int mute_state;
+		if (snd_mixer_selem_has_playback_switch(mixerElem)) 
+		{
+			snd_mixer_selem_get_playback_switch(mixerElem, SND_MIXER_SCHN_UNKNOWN, &mute_state);
+			if (!mute_state) // system Muted
+				return 0;
+		}
+		*/
```

Comment:
If disabled code is needed a comment would help.

Reply:
Comment added with why it was disabled and the commit that disabled it in the source. In all honesty, I'm not sure if this should be kept but I left it because it made it easier to read diffs at the time.